package/gargoyle/files/www/access.sh

```diff
@@ -129,6 +156,22 @@
 						<label id="disable_web_password_label" for="disable_web_password"><%~ DisablePassword %> <em>(<%~ warning %>)</em></label>
 					</span>
 				</div>
+
+				<div id="lockout_template" class="row form-group" style="display: none">
```

Comment:
I don't understand why we have this template and then other sections that display the message? Can you please clarify how this works? I only spotted this based on the JavaScript which suspiciously transfers the inner HTML one one element to another. 
If this is indeed what it is doing, please consider loading the text directly from JS to save characters. Translated text is still available in that space.

Reply:
The warnings can appear at three different places which caused it. Below the local web protocol, below the remote web control, and below the RFC 1918 checkbox. In case of the local web protocol, the reverse proxy help inside the warning template is displayed as well. I will move it into JS.

Reply:
Done.